octobot_trading/personal_data/trades/trade.py

```diff
@@ -84,10 +84,16 @@ def update_from_order(self, order, creation_time=0, canceled_time=0, executed_ti
         self.tag = order.tag
 
     def get_time(self):
-        return self.executed_time if self.status is not enums.OrderStatus.CANCELED else self.canceled_time
+        return (self.executed_time
+                if (self.status is not enums.OrderStatus.CANCELED
+                    and self.status is not enums.OrderStatus.EXPIRED)
+                else self.canceled_time)
```

Comment:
Shouldn't it be expired time if EXPIRED?

Reply:
could be, but as an expired order is just a canceled order that got automatically canceled by the exchange. I'd say we are good like that